tools/config/__init__.py

```diff
@@ -1058,9 +1062,29 @@ def validate_config(self):
 
         Arguments: None
         """
+
         params, _ = self.get_config_data()
+        err_msg = ""
+
+        for param in params:
+            min      = params[param].value_min
+            max      = params[param].value_max
+            accepted = params[param].accepted_values
+            value    = params[param].value
```

Comment:
With the 2nd suggestion for the for loop, these all become similar to:
```suggestion
            value    = param.value
```